app/assets/stylesheets/govuk-component/_govspeak.scss

```diff
@@ -20,6 +20,10 @@
     text-align: start;
   }
 
+  strong {
```

Comment:
Can this go into `govspeak/typography`? Specifically in [with em, sup, etc](https://github.com/alphagov/static/blob/master/app/assets/stylesheets/govuk-component/govspeak/_typography.scss#L187)?